libs/editor/WordPressEditor/src/main/java/org/wordpress/android/editor/GutenbergEditorFragment.java

```diff
@@ -98,17 +107,53 @@ private GutenbergContainerFragment getGutenbergContainerFragment() {
         return mRetainedGutenbergContainerFragment;
     }
 
+    public Bundle getTranslations() {
```

Comment:
This is a local method so we should change method access signature probably to private.

Reply:
This method name doesn't tell us enough information about what is the purpose of this method.
`getTranslations()` is too abstract I think ?